dotnetcore/NetstandardModuleSample/src/NetstandardModuleSample/DotNetSensorModule.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
```

Comment:
Please make sure for all the CS files, you only use one kind of indention style here: either 2 or 4.